### PR TITLE
Add backlog of changelogs from QMK Firmware

### PR DESCRIPTION
**Pull request by drashna**

### CODE REVIEW

File: _posts/2019-02-21-velocikey-match-rgb-animation-spped-to-typing-speed.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+layout: qmk-title
+title: "Velocikey: Match RGB animation speed to typing speed"
```
Comment (jackhumbert) [inline _posts/2019-02-21-velocikey-match-rgb-animation-spped-to-typing-speed.md]: Might have a typo in the filename here :)

File: _posts/2019-02-25-added-boardwalk-keyboard.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+layout: qmk-title
+title: "{Keyboard] Add boardwalk Keyboard"
```
Comment (noroadsleft) [inline _posts/2019-02-25-added-boardwalk-keyboard.md]:
```suggestion
title: "[Keyboard] Add Boardwalk Keyboard"
```

File: _posts/2019-03-11-keyboard-add-hotdox-keyboard.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+category: keyboards
+commit: 2a2f41c
+pr: 5365
+---
+
+We've added support for the [HotDox](https://shop.alpacakeyboards.com/) keyboard from [Alpaca Keyboards](https://www.alpacakeyboards.com/)!
+
+Check out this keyboard on the [QMK Configurator](https://config.qmk.fm/#/hotdox/LAYOUT_ergodox).
+
+You can also find the [Kono Store](https://kono.store/products/ergodox-hotdox).
```
Comment (noroadsleft) [inline _posts/2019-03-11-keyboard-add-hotdox-keyboard.md]:
```suggestion
You can also find the HotDox for sale on [Kono Store](https://kono.store/products/ergodox-hotdox).
```

File: _posts/2019-03-12-next-set-of-split-common-changes.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+layout: qmk-title
+title: "Next set of split_common changes"
+date: 2019-03-12 00:00:00 -0500
+category: core
+commit: 37932c2
+pr: 4974
+---
+
+THis updates a number of parts for the Split Keyboard code, to get it more ready for ARM Support (no, not here yet).
```
Comment (noroadsleft) [inline _posts/2019-03-12-next-set-of-split-common-changes.md]:
```suggestion
This updates a number of parts for the Split Keyboard code, to get it more ready for ARM Support (no, not here yet).
```

File: _posts/2019-03-12-next-set-of-split-common-changes.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+This PR: 
+
+* Eliminates the need for the RGB/Backlit dirty flags
+* Move over to i2c_master.c and i2c_slave.c for i2c transport (a large leap forward to ARM support)
+* Support greater than 8 columns for i2c based split keyboards
```
Comment (noroadsleft) [inline _posts/2019-03-12-next-set-of-split-common-changes.md]:
```suggestion
* Support greater than 8 columns for i2c-based split keyboards
```

File: _posts/2019-03-13-add-support-for-the60.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+title: "Add support for THE60"
+date: 2019-03-13 00:00:00 -0500
+category: keyboards
+commit: f9c5b80
+pr: 5385
+---
+
+We've added support for the Lazy Designer's [THE60](http://lazydesigners.cn) keyboard! 
+
+Check out this keyboard on the [QMK Configurator](https://config.qmk.fm/#/hotdox/LAYOUT_ergodox).
```
Comment (noroadsleft) [inline _posts/2019-03-13-add-support-for-the60.md]: Link address is wrong.

```suggestion
Check out this keyboard on the [QMK Configurator](https://config.qmk.fm/#/lazydesigners/the60/LAYOUT_split_spc_split_bspc).
```

File: _posts/2019-03-26-add-3-speed-mousekey-movement-options.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+title: "Add 3 speed mousekey movement option"
+date: 2019-03-29 00:00:00 -0500
+category: core
+commit: 346cbd8
+pr: 2246
+---
+
+This commit adds an optional Constant speed movement option for Mousekeys.
+
+This allows for a constant movement speed with Mousekeys may allow for more precise movement.
```
Comment (noroadsleft) [inline _posts/2019-03-26-add-3-speed-mousekey-movement-options.md]:
```suggestion
This allows for a constant movement speed with Mousekeys which may allow for more precise movement.
```

File: _posts/2019-04-11-port-direct-pins-from-split-common-matrix-to-matrix-c.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
+---
+layout: qmk-title
+title: "Port DIRECT_PINS from split_common/matrix.c to matrix.c"
+date: 2019-04-11 00:00:00 -0500
+category: core
+commit: 0137b02 
+pr: 5091
+---
+
+Adds support for Direct Pin connection keyboards (eg, no matrix, no diodes, just connected to the controller)
```
Comment (noroadsleft) [inline _posts/2019-04-11-port-direct-pins-from-split-common-matrix-to-matrix-c.md]: Grammar fixes.

```suggestion
Adds support for Direct Pin connection keyboards (e.g. no matrix, no diodes, just connected to the controller).
```

File: _posts/2019-04-12-add-altgr-ralt-support-to-send-string.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
+---
+layout: qmk-title
+title: "Add AltGr/RALT support to Send String"
+date: 2019-04-12 00:00:00 -0500
+category: core
+commit: a6857d1 
+pr: 4046
+---
+
+Adds support for AltGR, allowing for non-US keycodes in `SEND_STRING`
```
Comment (noroadsleft) [inline _posts/2019-04-12-add-altgr-ralt-support-to-send-string.md]: Missing a period.

```suggestion
Adds support for AltGR, allowing for non-US keycodes in `SEND_STRING`.
```

File: _posts/2019-04-19-oled-driver-feature.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+title: "OLED Driver Feature"
+date: 2019-04-19 00:00:00 -0500
+category: core
+commit: d326828
+pr: 5288
+---
+
+This adds a new OLED driver to QMK Firmware.
+
+This driver is smaller and faster, and includes features such as rotation of the OLED, or scroll an image
```
Comment (noroadsleft) [inline _posts/2019-04-19-oled-driver-feature.md]: Grammar/readability.

```suggestion
This driver is smaller and faster, and includes features such as rotation of the OLED, or scrolling an image.
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]
Looks good to me. :+1:

**Comment by drashna** [comment]
@jackhumbert ?

**Comment by jackhumbert** [comment]
Thanks :)

**Pull request merged**